im-crawler/src/main/java/com/wipro/ats/bdre/imcrawler/frontier/WorkQueues.java

```diff
@@ -44,11 +41,14 @@ public WorkQueues(String dbName, boolean resumable) {
         this.dbName = dbName;
         this.resumable = resumable;
         /*Hibernate autowire*/
-        ApplicationContext context = new ClassPathXmlApplicationContext("spring-dao.xml");
-        AutowireCapableBeanFactory acbFactory = context.getAutowireCapableBeanFactory();
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
         acbFactory.autowireBean(this);
     }
 
+    public Object execute(String[] params){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)